scripts/windows/msys2_prepare.bat

```diff
@@ -90,9 +90,7 @@ rem Remove breaking tools
 @echo Current time: %time%
 rem Upgrade the MSYS2 platform
 
-%BASH_BIN% -lc "%PACMAN% -y pacman"
-@echo Current time: %time%
-%BASH_BIN% -lc "%PACMAN% -u"
+%BASH_BIN% -lc "%PACMAN% -yuu"
```

Comment:
is the second `u` what's expected to fix the issue ?

Reply:
Magic!

Reply:
Yes. It performs a complete system upgrade instead of a single package upgrade.